README.md

```diff
@@ -71,6 +71,19 @@ IncogniaAPI api = IncogniaAPI.init("your-client-id", "your-client-secret");
 
 This will create a singleton instance of the IncogniaAPI class, which will handle token renewal automatically. You should reuse this instance throughout your application.
 
+The library also allow the users to configure the call timeout themselves. This will give them more control over the expected time response. This can be done by calling the init passing the CustomOptions object as a parameter.
+
+
+```java
+IncogniaAPI api = IncogniaAPI.init(
+    "your-client-id",
+    "your-client-secret",
+    CustomOptions.builder().timeout(20).build()
+);
+```
+
+If no parameter is passed the library will use the default timeout of 10 seconds of the OkHttp.
```

Comment:
```suggestion
If no parameter is passed the library will use the default timeout of 10 seconds.
```
Do you think we need to specify the use of OkHttp?

Reply:
No, we don't